nasa-gibs/worldview
# wv-2820 eslint

## Description
Our .eslintrc file has a lot of overrides. These should be turned back on and addressed. In this ticket, re-enable & address:

"no-use-before-define"

Fixes # wv-2820.

## How To Test

- git checkout wv-2820-eslint
- npm ci
- npm run watch
- Confirm that WV loads & runs correctly (E2E tests pass, so a quick look around should be sufficient)

Note that there is no **new** code here, these changes are related to code **ordering/organization**. 

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
